src/main/java/Task.java

```diff
@@ -0,0 +1,28 @@
+public class Task {
+    private final String description;
+    private boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public void markDone(){
+        this.isDone = true;
+    }
+
+    public void unmarkDone(){
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
```

Comment:
Is the method getStatusIcon necessary? It is a short line of code and seems to only be used in one method (see line 27)